### PR TITLE
fix: 🐛 Clear partialState when promise was rejected

### PR DESCRIPTION
**Pull request by Superstary**

### CODE REVIEW

File: packages/core/src/page/manager/AbstractPageManager.js [hunk 1]
```diff
@@ -327,7 +327,9 @@ export default class AbstractPageManager extends PageManager {
         extension.switchToStateManager();
         extension.clearPartialState();
       })
-      .catch(() => {});
+      .catch(() => {
+        extension.clearPartialState();
```
Comment (mjancarik) [inline packages/core/src/page/manager/AbstractPageManager.js:331]: Please, add test for current use case.

File: packages/core/src/page/manager/__tests__/AbstractPageManagerSpec.js [hunk 2]
```diff
@@ -416,6 +416,18 @@ describe('ima.core.page.manager.AbstractPageManager', () => {
...
     });
+
+    it('should clear partial state if resource is not loaded successfully', async () => {
+      spyOn(extensionInstance, 'load').and.returnValue(
+        Promise.resolve([Promise.reject()])
+      );
+      spyOn(extensionInstance, 'clearPartialState').and.stub();
+
+      await pageManager._getLoadedExtensionsState(controllerState);
+      await new Promise(resolve => setTimeout(resolve, 0));
```
[the remark this answers is not in the document]
Comment (Superstary) [reply packages/core/src/page/manager/__tests__/AbstractPageManagerSpec.js:427]: _getLoadedExtensionsState does not return a promise. Expect function is execute before this function is finished.

[the remark this answers is not in the document]
Comment (mjancarik) [reply packages/core/src/page/manager/__tests__/AbstractPageManagerSpec.js:427]: I think that `_getLoadedExtensionsState` returns promise as async function https://github.com/seznam/ima/blob/master/packages/core/src/page/manager/AbstractPageManager.js#L435.

[the remark this answers is not in the document]
Comment (Superstary) [reply packages/core/src/page/manager/__tests__/AbstractPageManagerSpec.js:427]: Sorry, I was wrong. _switchToPageStateManagerAfterLoaded don't have return statement and this is that problem.

**Pull request merged**